packages/nodejs/src/config/configmap.ts

```diff
@@ -61,6 +61,7 @@ export const JS_TO_RUBY_MAPPING: { [key: string]: string } = {
   debug: "debug",
   log: "log",
   logPath: "log_path",
+  logFilePath: "log_file_path",
```

Comment:
Is this line what adds it to the diagnose report as a config option? Because I don't think we should include it as a config option in the sent report. It's part of the paths section already and that contains more details.

What I see in a sent report:

<img width="928" alt="image" src="https://user-images.githubusercontent.com/282402/123973858-6a928b80-d9bc-11eb-9344-591775efd5f8.png">

Reply:
This line was added because the diagnose command uses the `JS_TO_RUBY_MAPPING` object to fetch the names for each field.
Because there was no name for the `logFilePath` field, it printed `undefined: '/tmp/appsignal.log'` before this change.
I'm pretty sure that'd mean this field was called `undefined` in the diagnose parser before, then.

In Ruby, the log_path and log_file_path aren't printed in the configuration at all.
I'll update the test in diagnose_tests to make sure this isn't the case in Node.js either, so we can remove this line again.